### ISSUE TITLE
Invalid types due to not found package reference

### ISSUE THREAD

**Issue by benmccann** [netlify/functions#415]
Rennovate bot is attempting to upgrade us to `@netlify/functions` v2 (https://github.com/sveltejs/kit/pull/10600), and this newest release looks like it might be broken as it causes our type checking to fail

**Describe the bug**

```
../../node_modules/.pnpm/@netlify+serverless-functions-api@1.7.0/node_modules/@netlify/serverless-functions-api/dist/cookie_store.d.ts:1:24 - error TS2307: Cannot find module '@netlify/node-cookies' or its corresponding type declarations.

1 import { Cookie } from '@netlify/node-cookies';
                         ~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in ../../node_modules/.pnpm/@netlify+serverless-functions-api@1.7.0/node_modules/@netlify/serverless-functions-api/dist/cookie_store.d.ts:1
```

**Configuration**

```
pnpm: 8.6.12 - ~/.local/share/pnpm/pnpm
```

**Comment by eduardoboucas** [comment]
Oops, my bad! This should be fixed in version 2.0.1. Could you please try?

**Comment by benmccann** [comment]
yes, it works now. thanks!!
